src/components/_global/BalChip/BalChip.vue

```diff
@@ -9,7 +9,7 @@ type Props = {
   label?: string;
   closeable?: boolean;
   size?: 'sm' | 'md' | 'lg';
-  color?: 'gray' | 'gradient' | 'white' | 'red' | 'orange';
+  color?: 'gray' | 'gradient' | 'white' | 'red' | 'orange' | 'orange-500';
```

Comment:
```suggestion
  color?: 'gray' | 'gradient' | 'white' | 'red' | 'orange' | 'orange';
```

Reply:
are you sure here? `| 'orange' | 'orange';`?

Reply:
Ahh sorry, skimmed that bit. Maybe we could rename it 'amber', or the original 'orange' could be named 'yellow' since it's lighter?